### PR TITLE
Update UG to fix peer review bugs 

### LINKED ISSUE #250: [PE-D] Edit is not in table of contents of User Guide
Missing `edit` hyperlink in table of contents!

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: ChuangZheQuan/ped#2

### LINKED ISSUE #251: [PE-D] Mix of British and American spelling
should standardise!

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: ChuangZheQuan/ped#3

### LINKED ISSUE #252: [PE-D] UG wrongly states interview Time format
"An INTERVIEW should follow the DateTime format yyyy-MM-dd, H:mm." is written in the UG. Time format should be HH:MM.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: wz27/ped#1

### LINKED ISSUE #264: [PE-D] Outdated description of 'show s/' command in Quick Start section of UG
Current 'show /s' command in 'Quick Start' section of the UG: `show s/` : Displays all unique expected salaries in RecruitIn, which would be 1 result that is `3000`.

Since the `3000` is no longer the first result in the product's current sample data, the team could update the description of this command to match the current scenario.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Preshita01/ped#4

### PR DESCRIPTION
**Pull request by kaushikkrdy**
## Changes 

* Fixes #250 by adding edit command to table of contents
* Fixes #251 by standardising British spelling 
* Fixes #252 by showing correct DateTime format for Interview 
* Fixes #264 by showing correct ShowCommand description in Quick Start

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -69,7 +71,8 @@ Scroll down to the bottom and click on `recruitIn.jar`.
 
     * **`delete`**`2` : Deletes the 2nd applicant shown in the list of all applicants, which would `Mary`.
 
-    * **`show`**`s/` : Displays all unique expected salaries in RecruitIn, which would be 1 result that is `3000`.
+    * **`show`**`s/` : Displays all unique expected salaries in RecruitIn, which would be 6 results that are
```
Comment (TohWeiJie01) [inline docs/UserGuide.md:74]: Would there not only be one unique salary shown since we added 2 applicants with s/3000 only?

**Review by TohWeiJie01 — COMMENTED** [review]
LGTM except for one section!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -203,7 +206,7 @@ a `.` unlike its preceding domain labels. (i.e. `John@u.sg` is valid)
     * TAG inputs such as `friends` and `colleagues` are allowed.
     * TAG inputs such as `best friends`, `old colleagues` and `seni@r` are not allowed.
 * ##### INTERVIEW `i/`
-  * An INTERVIEW should follow the DateTime format `yyyy-MM-dd, H:mm`.
+  * An INTERVIEW should follow the DateTime format `yyyy-MM-dd, HH:mm`.
```
Comment (TohWeiJie01) [inline docs/UserGuide.md:209]: I think Yao Yi edited this part as well for the date portion

File: docs/UserGuide.md (hunk 1 above)
Comment (kaushikkrdy) [reply docs/UserGuide.md:74]: I think there have been some changes to the model data, it now returns 6 unique salaries when I ran it just now.

**Pull request closed without merging**